README.md

```diff
@@ -15,78 +15,100 @@ This project is developed under Visual Studio 2022. Earlier versions of Visual S
 Installing the Package
 ======================
 
-Packages can be directly downloaded from [nuget.org](https://www.nuget.org/).
-
-It can also be downloaded using Visual Studio UI (Tools > NuGet Package Manager > Manage NuGet Packages for Solution and search for "Firebolt")
-
-Alternatively, packages can also be downloaded using Package Manager Console:
+Here is a FireboltNetSDK [NuGet page](https://www.nuget.org/packages/FireboltNetSDK/).
+- Install using **.NET CLI**
+```{r, engine='bash', code_block_name}
+dotnet add package FireboltNetSDK --version 0.*
+```   
+- Install using **Visual Studio UI**
+  - `Tools` > `NuGet Package Manager` > `Manage NuGet Packages for Solution` and search for `Firebolt`   
+- Install using **Package Manager Console**:
 ```{r, engine='bash', code_block_name}
-PM> Install-Package FireboltNetSDK -Version 0.0.1
+PM> Install-Package FireboltNetSDK -Version 0.*
 ```
 
 Examples
```

Comment:
I'd add statement that explains that the connector supports two types of authentication: user based and service account based (does it?). Otherwise these 2 examples may be a little bit confusing.